cloud/ovirt/ovirt_disks.py

```diff
@@ -238,6 +244,7 @@ def main():
         storage_domain=dict(default=None),
         profile=dict(default=None),
         format=dict(default=None, choices=['raw', 'cow']),
+        sparse=dict(default=None, type='bool'),
```

Comment:
On line 244 there is ```allocation_policy``` parameter, which I forgot to implement, which should handle this action. Can you please remove it?